rails_event_store_active_record/lib/rails_event_store_active_record/mappers/yaml.rb

```diff
@@ -0,0 +1,30 @@
+module RailsEventStoreActiveRecord
+  module Mappers
+    class YAML
+      def initialize
+        @serializer = ::YAML
+      end
+
+      def build_event_record(event)
```

Comment:
@paneq would you rename it to `event_to_record`, and `record_to_event`

Reply:
Better. I think I am in favor of 4 smaller methods to serialize and one (with 4 args) to deserialize, or we should introduce a closed object that is not Hash :) which would serve as transport layer between those 2 parts.

Reply:
I can't decide which one is better.

Reply:
I think with creating insteance of Event the contract is more cleaner now with `event_to_record`, and `record_to_event`

Reply:
@paneq please look again on code base, now looks cleaner and explaining behaviour in symmetric way